test/pom.xml

```diff
@@ -42,6 +41,17 @@
 		</repository>
 	</repositories>
 
+	<dependencies>
```

Comment:
Instead of adding dependencies, can you try adding the test scope to dependency management?